src/web/views/locales/en/unsuccessful/pending.html

```diff
@@ -0,0 +1,2 @@
+<p class="govuk-body">Your benefit status is pending</p>
+<p class="govuk-body">If you’re still having problems call 0300 330 7010 and select option 3.</p>
```

Comment:
Pending content hasn't been provided yet - I just added the phone number to the bottom of all decisions.